src/features/organizations/store.ts

```diff
@@ -46,6 +46,42 @@ const OrganizationsSlice = createSlice({
       state.eventsByOrgId[orgId] = remoteList(events);
       state.eventsByOrgId[orgId].loaded = new Date().toISOString();
     },
+    orgFollowed: (state, action: PayloadAction<ZetkinMembership>) => {
+      const membership = action.payload;
+
+      const existingMembership = state.userMembershipList.items.find(
+        (item) => item?.data?.organization.id === membership.organization.id
+      );
+
+      if (existingMembership?.data) {
+        existingMembership.data.follow = true;
+      } else {
+        const membershipWithId: ZetkinMembership & { id: number } = {
+          ...membership,
+          follow: true,
+          id: membership.organization.id,
+        };
+
+        state.userMembershipList.items.push(
+          remoteItem(membership.organization.id, { data: membershipWithId })
+        );
+      }
+
+      state.userMembershipList.loaded = new Date().toISOString();
```

Comment:
This implies that the _list_ has been loaded at this time, and I don't think that's a correct claim to make. The _item_ has been updated, so I think it's more valid to set the `loaded` property on the item.